features/FxA-I-11/README.md

```diff
@@ -5,50 +5,31 @@ Part of the [Eye Password Experiment](https://github.com/mozilla/fxa/blob/rfeele
 
 Idea in AHA: https://mozilla.aha.io/ideas/ideas/FXA-I-11
 
-## Outcomes
+## Problem Summary
 
-The success of this feature is being able to answer the question:
+There are a lot of errors related to password entry.
 
-"Is the 'Show' password button useful in its current form?".
-"Does it reduce the number of errors the users make on the change password page?"
-
-## Assumptions
+### Assumptions
 
 - Users use this functionality on Android more.
-- We believe the "Show" password button is useful.
-
-## Problem Statement
-
-The change password flow can be improved in the current system.
-
-## Hypothesis
-
-We believe that removing the unmasking functionality from both new and old password fields for people changing their passwords will achieve more errors and bounces.
-
-We will know this is true when we see an increase in bounces to the section, and Task Success errors outlined below.
-
-### Measure
+- The "Show" password button is useful.
```

Comment:
Some extra elaboration could be useful here?

`The assumption for this feature is that the "Show" password button is useful for reducing password entry errors. User's that have entered a password incorrectly might be more likely to click the "Show" password button on the second attempt.`